optimize/components/what-is-optimize.md

```diff
@@ -22,12 +22,8 @@ Geared toward business stakeholders, Optimize offers business intelligence tooli
 
 ![process performance dashboard](./img/dashboard-sharingPopover.png)
 
-As users run process instances through the server, Optimize makes REST API calls into the Camunda server, takes new historical data generated since the previous call, and stores the data in its own Elasticsearch database.
+As process instances run through the server, Optimize makes REST API calls to the Camunda server, collecting new historical data and storing it in its Elasticsearch database. This enables users to independently analyze reports and dashboards, gaining actionable insights without affecting runtime.
```

Comment:
Outside the scope of this ticket, but perhaps we should stop referring to Elasticsearch in our docs and instead just as the DB. With OpenSearch support, ES is just one of the implemented DBs available

Reply:
This is a great flag. I'll open an issue for my backlog.